print/components/PicassoPeriod.vue

```diff
@@ -92,28 +92,10 @@ export default {
     periodChunks() {
       const chunkSize = this.landscape ? 7 : 4
 
-      const start = dayjs.utc(this.period.start)
-      const end = dayjs.utc(this.period.end)
-      const hours = end.diff(start, 'hours')
-      const days = Math.floor(hours / 24) + 1
-
-      const numberOfChunks = Math.ceil(days / chunkSize)
-
-      if (numberOfChunks === 1) {
-        return [
-          {
-            start,
-            end,
-          },
-        ]
-      }
-
-      return [...Array(numberOfChunks).keys()].map((i) => {
-        return {
-          start: start.add(i * chunkSize, 'days'),
-          end: start.add((i + 1) * chunkSize - 1, 'days'),
-        }
-      })
+      return splitDaysIntoPages(this.period.days().items, chunkSize).map((chunk) => ({
```

Comment:
Yes, I agree. I went for the first option in [21715bb](https://github.com/ecamp/ecamp3/pull/3507/commits/21715bb102f129dcad7f3f9793de0025883c67fb). I like this one better because:
- the picasso splitting algorithm could potentially in the very far future rely on more advanced fields on the day entities
- the picasso splitting algorithm supports non-continuous periods (where not all days follow each other) or multi-period picassos this way (also options for the very far future)
- I feel like relying on the day indexes and finding days just by date is another obstacle on the potential way towards #2454. I might be wrong, but it's just another nudge in the direction of the other option for me.